app-i18n/libguess/libguess-1.2.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Character set guessing library"
+DESCRIPTION="Libguess is a high speed, open source character set encoding detector. \
+Libguess employs discrete-finite automata to deduce the character set of \
+the input buffer. The advantage of this is that all character sets can be \
+checked in parallel, and quickly."
```

Comment:
The first line is 85 chars long... You can rewrap DESCRIPTION like this (to have lines <= 80 chars):
```
DESCRIPTION="Libguess is a high speed, open source character set encoding \
detector. Libguess employs discrete-finite automata to deduce the character \
set of the input buffer. The advantage of this is that all character sets can \
be checked in parallel, and quickly."
```